routes/index.js

```diff
@@ -122,6 +121,8 @@ module.exports = () => {
 				const gameObj = {
 					game: true,
 					staffRole: account.staffRole || '',
+					isContributor: account.isContributor || '',
+					isTrialMod: account.isTrialMod || '',

```

Comment:
I don't know how your previous discussions of this topic went, but I think TrailMod probably needs to be a StaffRole. If hexi/coz has told you otherwise, then they are definitely the people to listen to. Contributor is fine like this though, and should not be a StaffRole.

Reply:
u might be right. Hexi said to do that for contributor and figured it might be good to try and do the same for trial mod as they dont get colors or special in game powers. they have very limited access and figured it might be easier to code them with that minimally added access as different variable all together? or is that wrong?

Reply:
seeing as the nature of their role is very different than AEM or contributors due to anonymity basis of role and limited access to staffRole functions that AEM share most alike. staffRole as far as i have seen is often a blanket and might be harder to rework with adding trial mod limits to the default assumed access. if that makes sense?

Reply:
I've had a look at the code. If you want to do it this way, it will work. My thinking was that StaffRole should be for anything with special powers, which TrailMods do have a few off. You will find a lot of `Boolean(userInfo.staffRole)) || TRIALMODS.includes(userInfo.userName)` and this would be an easy way to simplify that.

Reply:
ok ill revise to follow that format i suppose that makes sense

Reply:
although what about the places without the trialmod piece but that does have staffRole present id need to restructure those statements to remove all staffRole access to either purposely disclude trialmods or say only mods, editors, and admins..

Reply:
wouldnt making trialmod a separate variable resolve a lot of that extra work? since either way i will restructure anywhere that TRIALMODS and CONTRIBUTORS are used as those are the lists of hard coded users which the goal is to move over to db..?